deployments/kubernetes/chart/gitwebhookproxy/templates/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    fabric8.io/generated-by: {{ .Values.gitWebhookProxy.ingress.annotations.generatedBy }}
+    ingress.kubernetes.io/force-ssl-redirect: "{{ .Values.gitWebhookProxy.ingress.annotations.forceSslRedirect }}"
+    kubernetes.io/ingress.class: {{ .Values.gitWebhookProxy.ingress.annotations.ingressClass }}
+    monitor.stakater.com/enabled: "{{ .Values.gitWebhookProxy.ingress.annotations.enabled }}"
+  labels:
+{{ include "labels.stakater" . | indent 4 }}
+{{ include "labels.chart" . | indent 4 }}
+  name: {{ template "name" . }}
+  namespace: {{ .Values.gitWebhookProxy.namespace }}
+spec:
+  rules:
+  - host: {{ .Values.gitWebhookProxy.ingress.host }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ template "name" . }}
```

Comment:
take service name as a separate parameter it can differ from the ingress name

Reply:
Updated